TypeApprovalRegister+config.json

```diff
@@ -11,106 +11,300 @@
+                    },
+                    "port-reference": "/core-model-1-4:control-construct/logical-termination-point=tar-2-0-0-tcp-s-bm-001/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/local-port",
+                    "operation-reference": "/core-model-1-4:control-construct/logical-termination-point=tar-2-0-0-op-s-bs-004/layer-protocol=0/operation-server-interface-1-0:operation-server-interface-pac/operation-server-interface-capability/operation-name"
+                  }
                 }
               }
             },
             {
-              "uuid": "tar-1-0-0-application-p-0002",
-              "profile-name": "application-profile-1-0:PROFILE_NAME_TYPE_APPLICATION_PROFILE",
-              "application-profile-1-0:application-profile-pac": {
-                "application-profile-capability": {
-                  "application-name": "ExecutionAndTraceLog",
-                  "release-number": "1.0.0"
+              "uuid": "tar-2-0-0-action-p-002",
+              "profile-name": "action-profile-1-0:PROFILE_NAME_TYPE_ACTION_PROFILE",
+              "action-profile-1-0:action-profile-pac": {
+                "action-profile-capability": {
+                  "operation-name": "/v1/inform-about-application-in-generic-representation",
+                  "label": "API Documentation",
+                  "input-value-list": [
+                  ],
+                  "display-in-new-browser-window": true
                 },
-                "application-profile-configuration": {
-                  "approval-status": "application-profile-1-0:APPROVAL_STATUS_TYPE_APPROVED"
+                "action-profile-configuration": {
+                  "request": {
+                    "protocol-reference": "/core-model-1-4:control-construct/logical-termination-point=tar-2-0-0-tcp-s-bm-001/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/local-protocol",
+                    "address": {
+                      "ipv-4-address-reference": "/core-model-1-4:control-construct/logical-termination-point=tar-2-0-0-tcp-s-bm-001/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/local-address/ipv-4-address"
+                    },
+                    "port-reference": "/core-model-1-4:control-construct/logical-termination-point=tar-2-0-0-tcp-s-bm-001/layer-protocol=0/tcp-server-interface-1-0:tcp-server-interface-pac/tcp-server-interface-configuration/local-port",
+                    "operation-reference": "/core-model-1-4:control-construct/logical-termination-point=tar-2-0-0-op-s-bs-005/layer-protocol=0/operation-server-interface-1-0:operation-server-interface-pac/operation-server-interface-capability/operation-name"
```

Comment:
Can we also add an LTP instance for eatl-2-0-0-op-s-bs-005 (/docs) in config.json file

Reply:
Added **tar**-2-0-0-op-s-bs-005 in CONFIGfile.